mantle/cmd/kola/options.go

```diff
@@ -230,12 +232,18 @@ func syncOptionsImpl(useCosa bool) error {
 		if kola.Options.CosaWorkdir != "" && kola.Options.CosaWorkdir != "none" {
 			localbuild, err := sdk.GetLatestLocalBuild(kola.Options.CosaWorkdir)
 			if err != nil {
-				return err
+				if os.IsNotExist(errors.Cause(err)) && kola.QEMUOptions.DiskImage != "" {
+					fmt.Println("WARN: Unable to determine build metadata; continuing with provided disk image")
```

Comment:
```suggestion
					plog.Warning("Unable to determine build metadata; continuing with provided disk image")
```

Reply:
Do we actually need to warn for this? I see this code path as `kola` opportunistically trying to read metadata from the latest cosa build if there is one, but it's totally fine IMO if there isn't. We might not need it at all. Things which hard require cosa metadata, e.g. `--build ...` are handled higher up and do cause an error.

Reply:
>  I see this code path as kola opportunistically trying to read metadata from the latest cosa build if there is one, but it's totally fine IMO if there isn't. We might not need it at all. Things which hard require cosa metadata, e.g. --build ... are handled higher up and do cause an error.

This gets to why I tried to just circumvent this whole block by just checking for the `--qemu-image` argument.  I'm having trouble understanding why we would care about local metadata if we are providing a disk image.  Is there a case for using local metadata and the disk image at the same time?

Reply:
> Is there a case for using local metadata and the disk image at the same time?

Kinda? See `syncCosaOptions` for the "magic" we pull out of the cosa metadata. E.g. distro, Ignition version (less relevant now), and external tests (if for example you want to run an external test from `src/config/kola/tests` against a specific QEMU image).

The function is also meant to be generic over the platform in question. So it auto-fills the QEMU image on `qemu`/`qemu-unpriv`, but it can also auto-fill the live ISO on `qemu-iso` (and you can imagine the same thing AWS/GCP/etc... though those are still pending). So we want a generic fix for this instead of being specific to a platform. Hence why the specific check for `DiskImage` here looks out of place to me.

But again, the "cosa workdir with no local builds" is a corner-case and what you have here fixes it at least for the QEMU case, so WFM to just :ship: :it: as is and do any fixups as follow-ups. :)

Reply:
OK, thanks for the explanation!  I'll take one more pass at this and see if I something better comes out.